packages/snaps-cli/src/webpack/utils.ts

```diff
@@ -136,12 +136,12 @@ export async function getDefaultLoader({
        */
       module: {
         /**
-         * This tells SWC to output CommonJS modules. MetaMask Snaps
-         * doesn't support ES modules yet, so this is necessary.
+         * This tells SWC to output ES6 modules. This will allow Webpack to
+         * optimize the output code better.
```

Comment:
```suggestion
         * optimize the output code better.
         * The output will be transpiled to CommonJS later in the process.
```
WDYT about something like this? So we don't forget 😅